dot/config.go

```diff
@@ -162,6 +162,11 @@ type StateConfig struct {
 	Rewind uint
 }
 
+func (s *StateConfig) String() string {
+	return "" +
+		"rewind " + fmt.Sprint(s.Rewind)
```

Comment:
is there any reason for this empty string concatenating with the rest of the string? also change to: `fmt.Sprintf("rewind %d", s.Rewind)`

Reply:
It's just so adding fields won't create new deltas. I think originally this contained a state version flag, which I later removed, hence the form like this.

Reply:
But seems odd this way: `"" + "rewind " + fmt.Sprint(s.Rewind)`, is it possible change to `fmt.Sprintf("rewind %d", s.Rewind)` or at least `"rewind " + fmt.Sprint(s.Rewind)`?

Reply:
Done!